app/controllers/admins_controller.rb

```diff
@@ -64,9 +60,8 @@ def verify
     if voter.save
       send_email = params[:send_email] == "true"
       if send_email
-        # Will need to be replaced with deliver_now
         begin
-          UserMailer.voter_verified(voter, event_year).deliver
+          UserMailer.voter_verified(voter, event_year).deliver_now
```

Comment:
ok so for this PR, deliver_now does exist in rails 4.2 so this is fine.  Thanks again for bearing with me :)